3p/integration.js

```diff
@@ -29,7 +29,7 @@ import {adsense} from '../ads/adsense';
 import {adtech} from '../ads/adtech';
 import {doubleclick} from '../ads/doubleclick';
 import {twitter} from './twitter';
-import {register, run} from '../src/3p';
+import {register, run, checkData} from '../src/3p';
```

Comment:
we can turn it on, but right now it will complain about:

```js
class SomeClass {
  constructor() {
    /** @private */
    this.someProp_;
  }
}
```